backend/src/services/auth/custom-oauth-config.service.ts

```
@@ -2,9 +2,8 @@ import { Pool } from 'pg';
 import { DatabaseManager } from '@/infra/database/database.manager.js';
 import { SecretService } from '@/services/secrets/secret.service.js';
 import { AppError } from '@/api/middlewares/error.js';
-import { ERROR_CODES } from '@/types/error-constants.js';
 import logger from '@/utils/logger.js';
-import type { CustomOAuthConfigSchema } from '@insforge/shared-schemas';
+import { type CustomOAuthConfigSchema , ERROR_CODES } from '@insforge/shared-schemas';
```

ESLint

[error] Delete `·`

(prettier/prettier)